copy-app-conf.sh

```diff
@@ -9,4 +9,4 @@ else
   echo "Using existing $APP_CONF."
 fi
 
-grep 'TODO' "$APP_CONF" && die "Replace 'TODO' in $APP_CONF."
+grep 'TODO' "$APP_CONF" || echo "No TODOs in $APP_CONF"
```

Comment:
What motivates this? This script is used by both CI and, conceivably, new developers, and I worry about taking away the error message in the later usecase.

Reply:
This isn't a change to master, this is the already in place fix required to make `dev-start` work.